examples/app_test.py

```diff
@@ -0,0 +1,55 @@
+# pylint: disable=redefined-outer-name
+"""Tests for examples/app.y"""
+import json
+import pytest
+from falcon import testing
+import examples.app
+
+@pytest.fixture()
+def client():
+    """ client fixture """
+    return testing.TestClient(examples.app.run())
+
+def test_get_project_responses(client):
+    """Test get_project_responses"""
+    response = client.simulate_get('/page/get_project_responses')
+    assert response.status_code == 200
+
+    content = json.loads(response.content)
+
+    assert content
+    assert isinstance(content, list)
+
+    response = content[0]
+    # pylint: disable=line-too-long
+    assert list(response.keys()) == ['id', 'sequential_id', 'project_id', 'form_id', 'initial_response_id', 'pretty_id', 'submitted_at', 'responses', 'rating_aggregates', 'average_rating', 'num_ratings', 'created_at', 'updated_at', 'status', 'labels', 'responder_language', 'responder', 'deleted_at', 'submission_source']
+
+    assert isinstance(response['id'], int)
```

Comment:
any reason just checking instance on the id field?  what about the other fields?

Reply:
For these batch of tests, I am mostly just focusing on response id and the labels.  Will expand the tests as we build out more methods to test fields.